.github/workflows/run_tests.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ "ubuntu-latest", "macos-12" ]
+        os: [ "ubuntu-latest", "macos-12", "windows-latest" ]
```

Comment:
we can go with windows-latest, but then please also change the version in package.yml, so we have consistency between tests and packaging